core/src/main/scala/org/apache/spark/status/api/v1/api.scala

```diff
@@ -461,7 +461,7 @@ class ApplicationEnvironmentInfo private[spark] (
     val systemProperties: Seq[(String, String)],
     val metricsProperties: Seq[(String, String)],
     val classpathEntries: Seq[(String, String)],
-    val resourceProfiles: Seq[ResourceProfileInfo])
+    val resourceProfiles: collection.Seq[ResourceProfileInfo])
```

Comment:
It's a pity that not all cases can be changed,  because some of them involvepublic api changes, such as `systemProperties`, `metricsProperties` and `classpathEntries`, define them to `collection.Seq` need change the following function:

https://github.com/apache/spark/blob/1af0a510202bdadfbc1ab6d04b47fe01a23f4555/core/src/main/scala/org/apache/spark/util/Utils.scala#L2811-L2814